jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -455,6 +455,20 @@ def test_dynamic_update_slice(self, harness):
   def test_squeeze(self, harness: primitive_harness.Harness):
     self.ConvertAndCompare(harness.dyn_fun, *harness.dyn_args_maker(self.rng()))
 
+  @primitive_harness.parameterized(jtu.cases_from_list(
+      primitive_harness.lax_conv_general_dilated_general
+    ) + list(primitive_harness.lax_conv_general_dilated_known_limitations))
```

Comment:
@gnecula this is my idea for reducing the length of the tests; we only take a subset of this general tests, and make sure to run the cases with known limitations every time.

Reply:
I am thinking that we can avoid an exponential number of cases if we think of some features as orthogonal. For example, let's say that the precision feature is orthogonal to the others. Then instead of generating all the tests for all precisions, we generate a set of base tests and then every 3rd one get the HIGHEST precision. This reduces the tests by a factor of 3. Perhaps we can do that for a few more features. 

If you think in terms of code coverage, we replace path coverage for a sequence of conditionals with branch coverage: each branch is covered but not necessarily all combinations are covered.

Reply:
Oh I see! That may be indeed possible – and more desirable than taking a random subset of tests everytime, I think. I'll try to break it down like that.